optics-core/optics-core.cabal

```diff
@@ -77,6 +77,7 @@ library
                , indexed-profunctors    >= 0.1        && <0.2
                , transformers           >= 0.5        && <0.7
                , indexed-traversable    >= 0.1        && <0.2
+               , OneTuple               >= 0.3        && <0.4
```

Comment:
This looks like it might pull in quite a few extra dependencies to `optics-core`. Is that right, and can we avoid it?

Reply:
Actually, perhaps this is true only for older GHC versions?

Reply:
It looks to me like the package is quite careful to avoid excessive dependencies when possible.

Reply:
For psychological reasons I'd prefer us to not include this dependency.

The cost of not doing so is irrelevant - Solo can be copy pasted where `Identity'` was along with its Applicative instance.

Reply:
@arybczak Psychological reasons? Why put the compat code here when someone else has already written it there?

Reply:
> Psychological reasons? 

Yes, people don't like a lot of deps in "core" packages. Whether it's rational or not is irrelevant.

Reply:
Okay ... I'll copy whatever's needed over.